PowerEditor/src/functionList.xml

```diff
@@ -746,22 +746,23 @@
 						<nameExpr expr="\w+(?=[\(|:])" />
 					</className>
 					<function
-						mainExpr="(?&lt;=def\x20).+?(?=:)"
+						mainExpr="(?&lt;=\sdef\x20).+?(?=:)"
 					>
 						<functionName>
 							<funcNameExpr expr=".*" />
 						</functionName>
 					</function>
 				</classRange>
 				<function
-					mainExpr="(?&lt;=def\x20).+?(?=:)"
+					mainExpr="(?&lt;=\sdef\x20).+?(?=:)"

```

Comment:
Likewise.

Reply:
I think it is better to get the solution from
http://sourceforge.net/p/notepad-plus/discussion/482781/thread/515001cd/#b5d2
but it needs to improve the solutions suggested in that discussion. I'll comment in a one